sdk/test/opentelemetry/sdk/trace/export/noop_span_exporter_test.rb

```diff
@@ -7,4 +7,25 @@
 require 'test_helper'
 
 describe OpenTelemetry::SDK::Trace::Export::NoopSpanExporter do
+  export = OpenTelemetry::SDK::Trace::Export
+
+  let(:spans)    { [OpenTelemetry::Trace::Span.new, OpenTelemetry::Trace::Span.new] }
+  let(:exporter) { export::NoopSpanExporter.new }
+
+  it 'accepts an Array of Spans as arg to #export and succeeds' do
+    exporter.export(spans).must_equal export::SUCCESS
+  end
+
+  it 'accepts an Enumerable of Spans as arg to #export and succeeds' do
+    # An anonymous Struct serves as a handy implementor of Enumerable
+    enumerable = Struct.new(:span0, :span1).new
+    enumerable.span0 = spans[0]
+    enumerable.span1 = spans[1]
+
+    exporter.export(enumerable).must_equal export::SUCCESS
+  end
+
+  it 'accepts calls to #shutdown' do
+    exporter.shutdown
```

Comment:
Interesting to note that the no-op exporter doesn't return `FAILED_NOT_RETRYABLE` from `export` after `shutdown` has been called, which kinda violates the expectations for an Exporter. Maybe we should open an issue for that?

Reply:
https://github.com/open-telemetry/opentelemetry-ruby/issues/110

Reply:
I can handle that in a subsequent PR this (pacific) afternoon

Reply:
Hmm perhaps monday

Reply:
Monday monday https://github.com/open-telemetry/opentelemetry-ruby/pull/98/commits/e85a425c0226ae38ed5245f462b7e4fb5233a119